tests/components/homeassistant/test_scene.py

```diff
@@ -164,6 +165,51 @@ async def test_create_service(
     assert scene.attributes.get("entity_id") == ["light.kitchen"]
 
 
+async def test_delete_service(
+    hass: HomeAssistant, caplog: pytest.LogCaptureFixture
+) -> None:
+    """Test the delete service."""
+    assert await async_setup_component(
+        hass,
+        "scene",
+        {"scene": {"name": "hallo_2", "entities": {"light.kitchen": "on"}}},
+    )
+    await hass.services.async_call(
+        "scene",
+        "create",
+        {
+            "scene_id": "hallo",
+            "entities": {"light.bed_light": {"state": "on", "brightness": 50}},
+        },
+        blocking=True,
+    )
+
+    with pytest.raises(ServiceValidationError):
+        await hass.services.async_call(
+            "homeassistant",
```

Comment:
I remembered why we don't use the platform entity service helper for the `scene.create` service. We want to register the service under the `scene` domain and not under the `homeassistant` domain. We should probably stay consistent with that also for the scene delete service.